docs/release-procedure.md

```diff
@@ -23,29 +24,49 @@ as reviewing them again would not add a lot of value but would add lots of exces
 Hence non-cherry-pick commits should happen in a separate branch in your own fork, and
 be delivered to the release branch by using a PR.
 
-Note that Modin uses fully signed commits at least for releases, so you have to have GPG keys set up. See [onboarding instructions](https://github.com/modin-project/modin/blob/master/onboarding/onboarding.md) on where to get started.
+Note that Modin uses fully signed commits, so you have to have GPG keys set up. See [onboarding instructions](https://github.com/modin-project/modin/blob/master/onboarding/onboarding.md) on where to get started.
 
 To update Modin version, follow the instructions below.
 
-### Update the link to release in readme
+### Preparing the repo for a Major or Minor Version
 
-**Note**: from now on you work in a branch in your fork (in `origin`).
+**Note**: this should be done in your fork of Modin.
 
-Change the `README.md` in the repo root so release badge points to new (would be created) release.
+First, update your fork of Modin's master with the main repo's master. From your master, create a new
+branch called `release-X.Y.0` off of master. Create an empty commit in your new branch with the message
```

Comment:
we should pin badges in readme in this step, so it should not be an empty commit

Reply:
Sounds good to me!